src/main/scala/org/deepdive/ddlog/DeepDiveLogParser.scala

```diff
@@ -252,7 +252,7 @@ class DeepDiveLogParser extends JavaTokenParsers {
 
   def relationType: Parser[RelationType] =
     ( "like" ~> relationName ^^ { RelationTypeAlias(_) }
-    | rep1sep(columnDeclaration, ",") ^^ {
+    | "(" ~> rep1sep(columnDeclaration, ",") <~ ")" ^^ {
```

Comment:
I just wanted our syntax to be clearer and read better.  In fact, we could also say `like` is redundant because just using `R` is sufficient, but the function is not really defined only over R but the same types.  Let's leave it as it is if you disagree.